src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -137,30 +117,38 @@ public void testPeriodic() {
 	}
 
 	public void autonomousPeriodic() {
+		
+		double amountToGoLeft = Double.parseDouble(SmartDashboard.getString("DB/String 0", "1")); //Second value in feet
+		double amountToGoRight = Double.parseDouble(SmartDashboard.getString("DB/String 5", "1")); //Second value in feet
+		amountToGoLeft = drive.degreesToTicks(amountToGoLeft);
+		amountToGoRight = drive.degreesToTicks(amountToGoRight);
```

Comment:
The specific values we used don't matter anymore, this from autonomousPeriodic(). In fact, this type of thing should be in testPeriodic() in the future.

Reply:
I agree, test mode would be better for this sort of thing.

I don't think you want to keep this code. In fact, this is your test with degrees code. If you want to keep this code you probably want a "testWithDegrees()" or similarly named method. It would be nice to be able to switch back and forth from the angle to the distance tests.